bin/diff-interface-schemas.ts

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env ts-node-transpile-only
+#!/usr/bin/env npx ts-node-transpile-only
```

Comment:
It's not safe to use this syntax, as it's not supported everywhere - it gets interpreted as a single command:

```
/c/Users/G-Rath
❯ cat s.ts
#!/usr/bin/env npx ts-node-transpile-only

console.log('hello');

/c/Users/G-Rath
❯ ./s.ts
/usr/bin/env: ‘npx ts-node-transpile-only’: No such file or directory
```